docs/UserGuide.md

```diff
@@ -114,10 +114,18 @@ Cautions are placed in this guide to serve as warnings for certain actions.
 * Prefixes can be in any order.<br>
   e.g. If the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
+* If an input contains `"/"`, you cannot put spacing after its prefix.<br>
```

Comment:
```suggestion
* If an input contains `"/"`, you may wish to avoid putting spaces after `"/"`. This is to prevent the symbol from being recognized as a prefix.<br>
```